doc/design/resource_ownership_and_GC.md

```diff
@@ -0,0 +1,38 @@
+# Resource Ownership and GC
+
+## Resource Ownership
+
+An etcd cluster will create and manages etcd pods, services. Those resources like pods/services are owned by the etcd cluster.
+Two etcd cluster owns completely disjoint set of resources, even if they have the same name.
+For example, etcd cluster "A" was deleted and later etcd cluster "A" was created again; we think these are two different clusters.
+Thus, the new one won't be managing the resources from the old one, e.g. left as garbage.
```

Comment:
the new cluster should not mange the resources from the old one. the old resources should be treated as garbage and be collected.